charts/vault/README.md

```diff
@@ -0,0 +1,11 @@
+
+## Vault Helm Installation
+
+The dataset credentials are retreived from where they are stored (catalog/kubernetes secrets) using [Vault plugins](https://www.vaultproject.io/docs/internals/plugins).
+
+An example for such plugin is [vault-plugin-secrets-kubernetes-reader](https://github.com/mesh-for-data/vault-plugin-secrets-kubernetes-reader) which reads dataset credentials that are stored in kubernetes secrets.
+
+The Vault Helm Installation can support plugins for different configurations depending on the values provided to the chart.
+
+The values/ directory contains yaml files for the different plugins available.
```

Comment:
Is it possible to install multiple plugins together using what you have here?

Reply:
When using a single yaml for several plugins that works OK. if I use separate yaml files for each plugin and the use --values for each of them I could not install them properly. I might be doing something wrong so I continue to investigate.